src/css/base.scss

```diff
@@ -1673,6 +1677,14 @@ button.anchor {
     line-height: 20px;
 }
 
+.lh-32 {
+    line-height: 32px;
+}
+
+.lh-28 {
+    line-height: 28px;
```

Comment:
The same goes for line-heights. Move standard heights to base if need to be set explicitly.